src/controllers/DoctorLoadedPatientController.java

```diff
@@ -96,27 +96,39 @@ private Command DeletePatientPrescription() {
         };
     }
 
-/* PENDING IMPLEMENTATION IN PHASE 2
+
     private Command ViewPatientAppointments() {
         return (x) -> doctorView.viewAppointments(new AppointmentManager(getDatabase())
                 .getPatientAppointments(patientData));
     }
 
-    private Command getReport() {
-        return (x) -> new ReportManager(getDatabase()).getReportData(patientData);
+    private Command ViewPatientReports() {
+        return (x) -> {
+            ArrayList<ReportData> reportData = reportManager.getReportData(patientData);
+            if (reportData.size() > 0) {
+                doctorView.viewAllReports(reportData);
+                Integer targetReport = doctorView.viewReportPrompt();
+                doctorView.viewReport(reportData.get(targetReport - 1));
+            } else {
+                doctorView.noReports();
+            }
+        };
     }
 
-    private Command createReport() {
-        return (x) -> new ReportManager(getDatabase()).addReport(patientData, doctorData,
-                doctorView.reportDetailsPrompt().header(), doctorView.reportDetailsPrompt().body());
+    private Command CreatePatientReport() {
+        return (x) -> {
+            ReportDetails reportDetails = doctorView.reportDetailsPrompt();
+            reportManager.addReport(patientData, doctorData, reportDetails.header(), reportDetails.body());
+        };
     }
 
-    private Command deleteReport() {
+    private Command DeletePatientReport() {
```

Comment:
The user is not getting any feedback as to whether the deletion was successful or not.